_includes/footer.html

```diff
@@ -9,6 +9,9 @@
       <p class="mb-0">
         &gt;&gt;&gt; Dat > <a href="dat://eae17983cd6833287ab10ff4c1a3ca5d9ae7af7049eaa525912950720ece2e49/">dat://eae17983cd6833287ab10ff4c1a3ca5d9ae7af7049eaa525912950720ece2e49/</a>
       </p>
+      <p class="mb-0">
+        &gt;&gt;&gt; IPFS > <a href="https://ipfs.io/ipns/ipfs-website.ournetworks.ca/">https://ipfs.io/ipns/ipfs-website.ournetworks.ca/</a>
```

Comment:
Link `ipfs.infura.io`? ipfs.io is really unreliable.

Reply:
Looks like DNSlink and IPNS isn't supported on infura yet. Changing this over to the hash for now: https://github.com/ournetworks/ournetworks.ca/pull/133/commits/538f86da8452d74dbb4b04c1d3471e680b0d65b2#diff-7693eb643e8fe8b8d46b407909b408ceR13